app/controllers/conferences_controller.rb

```diff
@@ -15,26 +15,33 @@ def show
   end
 
   def create
-    team = current_user.student_team
     @conference = Conference.new(conference_params)
     @conference.season_id = current_season.id
-    @conference.gid = generate_gid(team)
+    @conference.gid = generate_gid(current_user)
 
     if @conference.save
-      redirect_to edit_team_path(team), notice: 'Conference was successfully created.'
+      redirect
     else
       render action: :new
     end
   end
 
   def redirect
```

Comment:
Having these edge cases based on the current user's role makes me wonder if we should not just have a dedicated controller for the orga namespace: `controllers/orga/conferences`... 🤔 

On the other hand the amout of logic here is still handleable... What do you think @juuh42dias ?

Reply:
We already have that controller. 💃  https://github.com/rails-girls-summer-of-code/rgsoc-teams/blob/master/app/controllers/orga/conferences_controller.rb

Reply:
Oh, should have looked at the repo 🙈  thanx a million @F3PiX 🙏 

So yeah, I guess we should opt for this controller for users coming from the orga views @juuh42dias ✌️

Reply:
I wonder if it makes sense to add a separate form in the orga namespace? So we don't have all the cancancan checks in the views. AFAIK one of the goals of the namespaces is getting independent of the cancancan checks. WDYallT?

Reply:
@klappradla I don't understand what you mean.

Reply:
Sorry @juuh42dias 🙈 
I just meant: the `ConferencesController` should not have any specific logic to distinguish between `admin` and normal users. `admin` users should instead use the controller in the `orga` namespace (plus the respective forms and views of course).